appveyor.yml

```diff
@@ -1,10 +1,10 @@
 ﻿image:
-    - Visual Studio 2017
     - Visual Studio 2019
 before_build:
     - nuget restore
 
 before_test:
     - nuget install xunit.runner.console -version 2.4.1
 test_script:
-    - xunit.runner.console.2.4.1\tools\net46\xunit.console.exe src\WpfMath.Tests\bin\Debug\net461\WpfMath.Tests.dll
+    - xunit.runner.console.2.4.1\tools\xunit.console.exe src\WpfMath.Tests\bin\Debug\net461\WpfMath.Tests.dll
+    - xunit.runner.console.2.4.1\tools\xunit.console.exe src\WpfMath.Tests\bin\Debug\netcoreapp3.0\WpfMath.Tests.dll
```

Comment:
I don't think you'll be able to run .NET Core tests with the cosole .NET runner. If you don't wanna, you can leave them as-is for now, I'll fix them myself later.